### PR TITLE
Add SUPREMM documentation for updating analytic help text and removing analytics in the efficiency tab. 

### PR DESCRIPTION
**Pull request by hltaylor26**
## Description
This PR adds documentation that provides users with instructions on how to edit the help text shown for each analytic in the efficiency tab and how to remove analytics.

## Motivation and Context
This change allows the efficiency tab to be customized as needed for different centers.

### CODE REVIEW

File: docs/analytics.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+---
+title: Customizing Help Text for Efficiency Analytics
+---
+
+This document describes how to customize the text displayed when viewing the analytics in the Efficiency Tab.
+
+**The automated upgade scripts do not have any support for preserving
```
Comment (jpwhite4) [inline docs/analytics.md:7]: Should we make the `etc/efficiency_anayltics.json` file `%config(noreplace)`?

File: docs/analytics.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+<figcaption>Figure 2. Example of CPU Usage scatter plot and corresponding help text.</figcaption>
+</figure>
+
+You may want to customize this text to provide more focused feedback for users at your center or provide links to help text as appropriate. To do so, please follow the directions provided below.
+
+**This customization will not be preserved if the Open XDMoD software is updated.**
+
+**These instructions only apply to Open XDMoD 10.0. This change will not work on earlier versions of Open XDMoD. For later
```
Comment (jpwhite4) [inline docs/analytics.md:35]:
```suggestion
**These instructions only apply to Open XDMoD 10.0. For later
```

File: docs/analytics.md (hunk 1 above)
Comment (hltaylor26) [reply docs/analytics.md:7]: That probably makes the most sense, I can't think of a reason not to.

File: docs/analytics.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
+---
+title: Customizing Help Text for Efficiency Analytics
+---
+
+This document describes how to customize the help text displayed when viewing the analytics in the Efficiency Tab and remove any analytics that are not pertinent to your center.
+
```
Comment (jpwhite4) [inline docs/analytics.md:6]:
```suggestion
These instructions are intended for a system administrator of an Open XDMoD instance. Information about the use of the efficiency tab can be found in the user manual which is available from the "Help" menu on the top right of the XDMoD portal.
```

File: docs/analytics.md (hunk 1 above)
Comment (hltaylor26) [reply docs/analytics.md]: After checking on this, `etc/efficiency_analytics.json` was added initially as `%config(noreplace)`, so this is already taken care of.

**Review by jtpalmer — APPROVED** [review]

**Pull request merged**